src/main/resources/avro/sequenceAnnotationmethods.avdl

```diff
@@ -0,0 +1,96 @@
+@namespace("org.ga4gh.methods")
+
+protocol SequenceAnnotationMethods {
+
+  import idl "methods.avdl";
+  import idl "sequenceAnnotations.avdl";
+
+  // FIXME: wiggle methods not defined.
+  
+  /******************  /features/search  *********************/
+
+  /**
+  This request maps to the body of `POST /features/search` as JSON.
+  */
+  record SearchFeaturesRequest {
+    /**
+    The annotation sets to search within. Either `featureSetIds` or
+    `parentIds` must be non-empty.
+    */
+    array<string> featureSetIds = [];
+
+    /**
+    Restricts the search to direct children of the given parent `feature`
+    IDs. Either `featureSetIds` or `parentIds` must be non-empty.
+    */
+    array<string> parentIds = [];
+
+    /**
+    If specified, this query matches only annotations which overlap this range.
+    */
+    union { null, QueryRange } range = null;
+
+    // TODO: Fix this field. Clarify semantics around how OntologyTerm matching works, or
+    // change the Ontology term field on the feature.
```

Comment:
I will merge the PR with master, since sequence-annotation #246 has been merged into master.